### PR TITLE
fixed broken link

### LINKED ISSUE #41967: [it] Broken link for "Create a cluster in High-Availability" in Kubernetes Components page
From https://kubernetes.io/it/docs/concepts/overview/components/ the link to 
https://kubernetes.io/docs/admin/high-availability/  is broken

### PR DESCRIPTION
**Pull request by deepak4566**
fixes #41967 
modified broken link as the link is broken in every segment

### CODE REVIEW

**Comment by Fale** [comment]
This change is not correct. URLs should be relative, not absolute!

**Comment by mattiaperi** [comment]
/remove-approve
as per @Fale comment

**Comment by deepak4566** [comment]
@Fale  I didn't find the file of high availability and the links broken in every segment  so I used absolute link  
256877889-c65fe8c8-2342-4de3-a6f6-09615fd91acd

**Comment by utkarsh-singh1** [comment]
Hi @deepak4566, you can raise another PR with the changes mentioned, which can also rebuild the netlify CI tool.

**Comment by Fale** [comment]
@deepak4566 the fact the file is there or not is irrelevant on the absolute vs relative URLs.
A relative URL gets compiled to an absolute one by the browser picking up the current domain and appending to it the relative path.
If you are linking to the same domain, a relative URL and an absolute URL will behave in the same way.

Now, on the point that the file does not exists, you have to consider that Kubernetes' website uses Hugo with a lot of preprocessing tasks, so even if a file does not exist, a URL can still work. To ensure that everything works, we use a test deploy on netlify so that we can check exactly how the whole things gets rendered. For some reason it failed the CI, but it might be a transient error.

**Comment by deepak4566** [comment]
@Fale understood 😃

File: content/it/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Questo è un diagramma di un cluster Kubernetes con tutti i componenti e le loro
 I componenti del Control Plane sono responsabili di tutte le decisioni globali sul cluster (ad esempio, lo scheduling) oltre che a rilevare e rispondere agli eventi del cluster (ad esempio, l'avvio di un nuovo {{< glossary_tooltip text="pod" term_id="pod">}} quando il valore `replicas` di un deployment non è soddisfatto).
 
 I componenti della Control Plane possono essere eseguiti su qualsiasi nodo del cluster stesso. Solitamente, per semplicità, gli script di installazione tendono a eseguire tutti i componenti della Control Plane sulla stessa macchina, separando la Control Plane dai workload dell'utente.
-Vedi [creare un cluster in High-Availability](/docs/admin/high-availability/) per un esempio di un'installazione multi-master.
+Vedi [creare un cluster in High-Availability](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/high-availability/) per un esempio di un'installazione multi-master.
```
Comment (dipesh-rawat) [inline content/it/docs/concepts/overview/components.md:31]:
```suggestion
Vedi [creare un cluster in High-Availability](/docs/setup/production-environment/tools/kubeadm/high-availability/) per un esempio di un'installazione multi-master.
```

Comment (deepak4566) [reply content/it/docs/concepts/overview/components.md:31]: Hey thanks for suggestion 👍 will commit it

**Comment by dipesh-rawat** [comment]
@deepak4566 Looks like the Netlify build failed. Could you push an empty commit to trigger a rebuild using:
`git commit --amend --no-edit && git push --force-with-lease`

**Comment by deepak4566** [comment]
@dipesh-rawat hey can u guide to squash  my commits , due to some error in command the head 20 commits of master got added

**Comment by deepak4566** [comment]
@Fale @dipesh-rawat  should i need to open new pr ?

**Comment by Fale** [comment]
to get rid of the additional commits, you can do a git rebase

As for the new commit, you should create different PRs every time you are doing something logically different or for changes cross languages, since different people will need to review those changes and it makes it easier to review the history if changes are kept small

**Comment by deepak4566** [comment]
> to get rid of the additional commits, you can do a git rebase
> 
> As for the new commit, you should create different PRs every time you are doing something logically different or for changes cross languages, since different people will need to review those changes and it makes it easier to review the history if changes are kept small

@Fale hey got rebase isn't working ,can say what commands could be executed from here

**Comment by Fale** [comment]
It seems like the addition of a conflicting commit is the problem. We can do a cherrypick to solve this:

1. git checkout deepak_1
2. git checkout -b deepak_1_old
3. git branch -D deepak_1
4. git checkout main
5. git checkout -b deepak_1
6. git cherry-pick 20636627879a539f19508106454dcf9316aebaf8
7. git push -f origin deepak_1 (substitute `origin` for your origin name)

**Comment by deepak4566** [comment]
> It seems like the addition of a conflicting commit is the problem. We can do a cherrypick to solve this:
> 
> 1. `git checkout deepak_1`
> 2. `git checkout -b deepak_1_old`
> 3. `git branch -D deepak_1`
> 4. `git checkout main`
> 5. `git checkout -b deepak_1`
> 6. `git cherry-pick 20636627879a539f19508106454dcf9316aebaf8`
> 7. `git push -f origin deepak_1` (substitute `origin` for your origin name)

@Fale thanks you very much for these trobleshooting steps , i was really confused at a time ,thanks very much for helping me out

**Comment by sftim** [comment]
/hold

OK to unhold once the v1.28 release has happened

**Pull request merged**